app/modules/import-export/module.php

```diff
@@ -171,16 +171,16 @@ private function render_import_export_tab_content() {
 			if ( ! empty( $penultimate_imported_kit ) ) {
 				$revert_text = sprintf(
 					esc_html__( 'Remove all the content and site settings that came with "%1$s" on %2$s %3$s and revert to the site setting that came with "%4$s" on %5$s.', 'elementor' ),
-					! empty( $last_imported_kit['kit_name'] ) ? $last_imported_kit['kit_name'] : esc_html__( 'imported kit', 'elementor' ),
+					! empty( $last_imported_kit['kit_title'] ) ? $last_imported_kit['kit_title'] : esc_html__( 'imported kit', 'elementor' ),
```

Comment:
maybe add a fallback to the kit name? (probably need to check with the product)

`kit_title ?? kit_name ?? "imported kit"`

and then extract to some reusable method (`extract_kit_title()` or something?)

Reply:
These were the specifications of the task. The name is dashed and they don't want it to be used here and this decision should not be part of this PR anyway